### ISSUE TITLE
Sorry, your LaTeX code couldn't compile for some reason. Please check the errors below for details, or view the raw log.

### ISSUE THREAD

**Issue by khawar512** [posquit0/Awesome-CV#353]
I have simply download the below project from Github and upload it in overleaf. After uploading, it shows different errors.

https://github.com/ravijo/Awesome-CV

![image](https://user-images.githubusercontent.com/11488932/105992936-0d329f80-60e9-11eb-9ee0-a3d78a29af23.png)

**Comment by rjhefley** [comment]
Greetings Khawar,
I had the same issue when I downloaded the project from Github as well. I actually downloaded the project, and put it into VSCode. I had a massive amount of errors as well.

The solution that worked for me was by going to the Overleaf project found here: https://www.overleaf.com/latex/templates/awesome-cv/dfnvtnhzhhbm 

It appears that it builds correctly on this link. But my solution consisted of rebuilding the entire folder structure in VSCode and copy paste the LaTeX code from Overleaf and it builds correctly in VSCode now.

Hope this helps you get started!!

Regards,
Ryan
